bot.py

```diff
@@ -387,19 +387,19 @@ def start_game(bot, update, args, job_queue):
             def send_first():
```

Comment:
Line 386 - ``TelegramDeprecationWarning: The @run_async decorator is deprecated. Use the `run_async` parameter of your Handler or `Dispatcher.run_async` instead``